pages/index.js

```diff
@@ -1,51 +1,51 @@
 import styled from "styled-components";
-import DonationListItem from "@/components/DonationListItem";
-
-export default function HomePage({ donations, myUserID, handleEditDonation }) {
-  const filteredDonations = donations.filter(
-    (donations) => donations.userID != myUserID
-  );
+import logoHomepage from "../assets/logoHomepage.png";
+import Image from "next/image";
 
+export default function HomePage() {
   return (
-    <>
-      <h2> Spenden in deiner Nähe: </h2>
-      <StyledList>
-        {filteredDonations.map((donation) => {
-          return (
-            <DonationListItem
-              key={donation.id}
-              donation={donation}
-              donations={donations}
-              handleEditDonation={handleEditDonation}
-              myUserID={myUserID}
-            />
-          );
-        })}
-      </StyledList>
-
-      <StyledNavigation as="a" myUserID={myUserID} href={"/myDonations"}>
-        Ich möchte spenden!
-      </StyledNavigation>
-    </>
+    <StyledDiv>
```

Comment:
use something like "Wrapper" for naming a styled div :)

Reply:
change it to wrapper :)